certificate-authentication.md

```diff
@@ -55,9 +55,9 @@ It is recommended that you use [OpenSSL](https://www.openssl.org/) to create key
     ```bash
     Country Name (2 letter code) [AU]:US
     State or Province Name (full name) [Some-State]:California
-    Locality Name (eg, city) []:San Francisco
-    Organization Name (eg, company) [Internet Widgits Pty Ltd]:PingCAP Inc.
-    Organizational Unit Name (eg, section) []:TiDB
+    Locality Name (e.g. city) []:San Francisco
+    Organization Name (e.g. company) [Internet Widgits Pty Ltd]:PingCAP Inc.
+    Organizational Unit Name (e.g. section) []:TiDB
```

Comment:
These lines are within the code block which Vale does not process, so I leave "e.g." as it is and unify the format. Same for other changes in this file.

Reply:
Got it.